cdm/core/src/test/java/ucar/nc2/util/cache/TestNetcdfFileCache.java

```diff
@@ -68,8 +68,9 @@ private void loadFilesIntoCache(File dir, FileCache cache) {
   @Test
   public void testNetcdfFileCache() throws IOException {
     loadFilesIntoCache(new File(TestDir.cdmLocalTestDataDir), cache);
-
     cache.showCache(new Formatter(System.out));
+    assertThat(cache.hits.get()).isEqualTo(0);
+    assertThat(cache.miss.get()).isEqualTo(count);
```

Comment:
Did I correctly notice that you are all moving away from using `assertThat` now? Should I update my tests?

Reply:
Actually we are moving to only using the google truth asserts, so this is good as it.